components/mdx/Strong.js

```diff
@@ -0,0 +1,7 @@
+const Strong = ({children}) => {
+    return (
+        <span className=" font-bold">{children}</span>
```

Comment:
I don't think the extra space is needed here

```suggestion
        <span className="font-bold">{children}</span>
```